src/utils/wagmi.ts

```diff
@@ -48,7 +51,9 @@ const connectors = connectorsForWallets([
       trustWallet({ chains }),
     ],
   },
-])
+])().concat([ledgerConnector])
+
+console.log("connectors:", connectors())
```

Comment:
Can be removed, unless it makes sense to keep it for now for testing.

Reply:
Done